src/Authorization.AspNetCore/GraphQlBuilderExtensions.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using GraphQL.Server.Authorization.AspNetCore;
+using GraphQL.Validation;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace GraphQL.Server
+{
+    public static class GraphQLBuilderExtensions
+    {
+        /// <summary>
+        /// Adds the GraphQL authorization.
+        /// </summary>
+        /// <param name="builder">The GraphQL builder.</param>
+        /// <returns></returns>
+        public static IGraphQLBuilder AddGraphQLAuthorization(this IGraphQLBuilder builder)
+        {
+            builder
+                .Services
```

Comment:
We should add this line in these methods to make sure `IHttpContextAccessor` is registered. It is not by default.

``` csharp
builder.Services.TryAddSingleton<IHttpContextAccessor, HttpContextAccessor>();
```